frontend/src/components/TaskCalendar.jsx

```
@@ -0,0 +1,513 @@
+            >
+              <option value="Low">Low Priority</option>
+              <option value="Medium">Medium Priority</option>
+              <option value="High">High Priority</option>
+            </select>
+          </div>
+          <div className={styles.popupActions}>
+            {selectedTask.status === "Completed" ? (
+              <button
+                className={styles.actionButton}
+                onClick={handleMarkNotDone}
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)